ngraph/core/src/pass/visualize_tree.cpp

```diff
@@ -537,6 +538,12 @@ string pass::VisualizeTree::get_node_name(shared_ptr<Node> node)
         rc += "\\n" + (nvtmn ? string("name: ") : "") + node->get_name();
     }
     rc += "\\n" + (nvtmn ? string("type_name: ") : "") + std::string(node->get_type_name());
+    if (nvtmn && is_type<ngraph::opset1::FakeQuantize>(node))
+    {
+        rc += "\\nlevels: " +
+              std::to_string(
+                  std::dynamic_pointer_cast<ngraph::opset1::FakeQuantize>(node)->get_levels());
+    }
```

Comment:
This function returns node name, why we insert some specific node attributes?
To serialize attributes we need to use AttributesVisitor API and introduce new mode inside VisualizeTree to dump operation attributes.

Reply:
was removed, thanks